front/src/components/HeaderMenu/HeaderMenuContainer.tsx

```diff
@@ -18,63 +18,63 @@ function HeaderMenuContainer() {
     return [
       {
         text: "Statistics",
-        requiredAbp: [],
+        requiredAbps: [["view_inventory", "view_shipments", "view_beneficiary_graph"]],
```

Comment:
Why is it required to define the same ABPs in two places?

Reply:
We have a ticket for this already.

https://trello.com/c/m0JT1wpo